snap/gui/org.freecadweb.FreeCAD.svg

```diff
@@ -0,0 +1,280 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
```

Comment:
That's the FreeCAD logo with an "RT" added, so I can distinguish it in the launcher. It's just a WIP idea, however. Some kind of "RT" badge with an opaque background might be a better fit.